image-rs/tests/credential.rs

```diff
@@ -19,7 +19,7 @@ pub mod common;
 #[tokio::test]
 #[serial]
 async fn test_use_credential(#[case] image_ref: &str, #[case] auth_file_uri: &str) {
-    common::prepare_test().await;
+    common::prepare_test("aa-offline_fs_kbc-resources.json").await;
```

Comment:
I think it would be better to declare this as a `const`, or we add the parameter to `#[case]` macro. How do you like?

Reply:
Done. I went with adding some constants to `common/mod.rs` as the signature verification tests do not use the `#[case]` macro.